### PR TITLE
[docker] Fixes #1514 Install tig and tig-explorer inside skycoin-dev image

### LINKED ISSUE #1514: Install tig inside skycoin-dev image
[tig](https://github.com/jonas/tig) is a very useful text-mode interface for `git`, including repository browser , and can be integrated with `vim` .

### PR DESCRIPTION
**Pull request by stdevNorge**
Fixes #1514 

Changes:
- Install tig and a vim plugin(tig-explorer) for it

Does this change need to mentioned in CHANGELOG.md?
Yes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Add `uxouts` to `POST /api/v1/wallet/transaction`, to allow specific unspent outputs to be used in a transaction.
 - Add Dockerfile in docker/images/dev-cli to build a docker image suitable for development.
 - Coin creator tool, `cmd/newcoin`, to quickly bootstrap a new fiber coin
+- Install a text-mode interface for git (tig) and a vim plugin for it.
```
Comment (olemis) [inline CHANGELOG.md]: where ?

**Comment by gz-c** [comment]
No files changed?

**Comment by olemis** [comment]
@gz-c [changes](https://github.com/skycoin/skycoin/blob/develop/docker/images/dev-cli/Dockerfile#L185-L188) are already in `develop` branch. Duplicate PR ?

**Comment by olemis** [comment]
I am not sure of what happened ... but this PR should be closed . @gz-c feel free to do so

**Pull request closed without merging**